### DISCUSSION TITLE
Backlinks are dead. And Breadcrumbs has killed them. 👊🏼

### DISCUSSION THREAD

**Discussion by scriptor-dga** [SkepticMystic/breadcrumbs#264] [Ideas]
Am I wrong to assume that, if we were to use Breadcrumbs systematically in every note we create, backlinks would become useless? I see backlinks as a messy family dinner where everyone talks over each other, whereas with Breadcrumbs you can neatly distinguish between parents, siblings and children of each note. Also, you don't need to manually update a Map of Content (which is also limited because each child can only have one parent) because the tree is semi-automatically created for you - just add at least a parent and a next/prev/same to each note - and it's always in the foreground when the note is open. The seamless navigation in the tree in all directions reminds me of web browsing, but with my own files. And if you really need to export the MOC in a note, you can do it at any level/direction/depth you want. Using Breadcrumbs made using Obsidian a lot more fun and satisfying. It might have even brought Notion's database functionalities (the only pro that Notion still has) into Obsidian in its own way but I'll have to think about it.
I think I will do a video about Breadcrumbs after the ["implied relationships detection"](https://github.com/SkepticMystic/breadcrumbs/issues/259) will be improved. That will be a huge step forward IMHO, as it'll make the whole threading experience even more fun and seamless. 💪🏼

**Comment by Brettelectric** [comment]
I have an MOC structure, and I used to use the backlinks pane to click 'up' the tree, but the larger my vault gets, the more crosslinks I generate (which is the point of something like Obsidian), which means that my backlinks pane has heaps of notes in it. It becomes hard to find the backlink to take me back up one level.

With Breadcrumbs though, I can specify which links make up my hierarchy, and leave the crosslinks as they are, so it's easy to find the link to go 'up' the tree.

I still use the backlinks pane sometimes, to discover which non-hierarchical links are coming into the current note.
